docs/getting-started/roadmap.rst

```diff
@@ -1,3 +1,25 @@
 =======
 Roadmap
-=======
\ No newline at end of file
+=======
+
+Here will be recorded the roadmap for the future development of PyMedPhys.
+
+
+Graphical User Interface (GUI)
+------------------------------
+
+A goal of the graphical user interface is to have it customisable, and
+expandable. It is envisioned that the not yet released
```

Comment:
I've noticed this style of writing quite a few times as I've perused the docs. Active verbs form more efficient, powerful statements. They hold interest. Phrases like:

"**We envision** that the..."

should be preferred over

"**It is envisioned** that the..."

Also, this isn't an academic paper, *don't be afraid to use pronouns*. Heck, business/academia's over-insistence on avoiding them in favour of "it" constructions is a travesty of quality English prose.

Reply:
:) I think I've been tainted. I'm too stuck in not being able to use pronouns.